### ISSUE TITLE
LoginAttempt written for every controller in stack when using BasicAuth

### ISSUE THREAD

**Issue by mikey-harveycameron** [silverstripe/silverstripe-framework#10832]
## Affected Version

SS4.13

## Description

The LoginAttempt table is written to by every controller when the site is accessed using BasicAuth. This includes Elemental block controllers. The LoginAttempt table quickly becomes massive and the site slows down.

Is it necessary to record BasicAuth attempts? If so, can we restrict to the top controller? And only once per session?

## Steps to Reproduce

1. Add lots of elemental blocks to a page.
2. SS_USE_BASIC_AUTH="ADMIN"
3. Reload the page a few times.
4. Marvel at your bloated database.

**Comment by maxime-rainville** [comment]
I don't think that's default behaviour. [`Security::login_recording`](https://github.com/silverstripe/silverstripe-framework/blob/0c40cc9b3d9658ad868c560059c88255d2ee0674/src/Security/Security.php#L164) is to `false` by default.

**Comment by mikey-harveycameron** [comment]
> I don't think that's default behaviour. [`Security::login_recording`](https://github.com/silverstripe/silverstripe-framework/blob/0c40cc9b3d9658ad868c560059c88255d2ee0674/src/Security/Security.php#L164) is to `false` by default.

This is correct, but it was also made dependent on max login attempts, which is set to 10, so it is never false!
